### DISCUSSION TITLE
How to create dynamic length columns ?

### DISCUSSION THREAD

**Discussion by futahei** [pdfme/pdfme#93] [Q&A] [answered]
Hello. Thank you for the great library!

How can I use pdfme to arrange dynamic length items?
I need to do this when creating an invoice for example.

**Comment by hand-dot** [comment, marked as answer]
Hi @futahei 

pdfme doesn't have a dynamic length table feature.
However, it is possible by dynamically creating a template.

**Comment by hand-dot** [comment]
ref: https://github.com/pdfme/pdfme/discussions/148
